### ISSUE TITLE
onError didn't give me http response

### ISSUE THREAD

**Issue by lovepanda603** [Imgkl/EventFlux#25]
server: user springboot，if There is an error in the business in the program，my program will return a 500 http status,and a json body："{'code':1001,'msg':'no permit'}"

onError: (oops) {
        // Oops! Time to handle those little hiccups.
        // You can also choose to disconnect here
        print("---onError----");
        print(oops.message);
      }
print：
Connection Error Status:500, Connection Error Reason:
can't get 这body "{'code':1001,'msg':'no permit'}"

**Comment by Imgkl** [comment]
@lovepanda603 Thanks for filing the issue, Are you saying when there is a error, the status code and the json body is not getting printed by doing this?

````
onError: (oops) {
// Oops! Time to handle those little hiccups.
// You can also choose to disconnect here
print("---onError----");
print(oops.message);
}
````

**Comment by lovepanda603** [comment]
> @lovepanda603 Thanks for filing the issue, Are you saying when there is a error, the status code and the json body is not getting printed by doing this?
> 
> ```
> onError: (oops) {
> // Oops! Time to handle those little hiccups.
> // You can also choose to disconnect here
> print("---onError----");
> print(oops.message);
> }
> ```

my print is :  I/flutter ( 5145): Connection Error Status:500, Connection Error Reason: 

I can't get nothing,but Program business requirements zhe json

**Comment by Imgkl** [comment]
Hey @lovepanda603,

This is fixed on the latest version, `2.1.2`. Please update the package to get this data. 

Now the `EventFluxException` has message prop for you to get this.
